### PR TITLE
Feat #1763/remove unused libs

### LINKED ISSUE #1763: [FEATURE] Cleanup project dependencies
**Describe how this feature will help us improve**
Describe why do you think this feature is needed. 

`package.json` contains dependencies that are not used anymore (like `vuepress` or I think `axios`). Some depencies can possibly be moved to devDependencies. Also some dependencies are used only in one case (i.e. `vue-fragment`) so they can be replaced.

**Is your proposal related to a problem?**
Describe the issue or type its number

**Describe the solution you'd considered**
A concise description of your preferred solution. Things to address include:

  - Details of the technical implementation
  - Tradeoffs made in design decisions
  - Caveats and considerations for the future

If there are multiple solutions, please present each one separately. Save comparisons for the very end.

**Additional context**
Anything you would like to add but was not sure which section it fits.

### PR DESCRIPTION
**Pull request by Baroshem**
# Related issue

Closes #1763 

# Scope of work

Removed some unnecessary dependencies. Decided not to remove the body-scroll-lock as it only takes about 1 KB and is really useful. Removing other dependencies like simplebar (20kb) is bigger profit for us.

# Screenshots of visual changes

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/molecules/SfScrollable/SfScrollable.vue [hunk 1]
```diff
@@ -51,7 +48,7 @@ export default {
     return {
       isHidden: true,
       hasScroll: false,
-      contentEl: undefined,
+      // contentEl: undefined,
```
Comment (justyna-13) [inline packages/vue/src/components/molecules/SfScrollable/SfScrollable.vue:51]: do we need this comment? :)

File: packages/vue/src/components/molecules/SfScrollable/SfScrollable.vue [hunk 2]
```diff
@@ -64,22 +61,22 @@ export default {
...
-    this.$nextTick(() => {
-      this.contentEl = this.$refs.content.$el.querySelector(
-        ".simplebar-content"
-      );
-      if (typeof MutationObserver === "undefined" || !this.contentEl) return;
-      const observer = new MutationObserver(this.sizeCalc);
-      this.sizeCalc();
-      observer.observe(this.contentEl, { childList: true });
-    });
+    // this.$nextTick(() => {
```
Comment (justyna-13) [inline packages/vue/src/components/molecules/SfScrollable/SfScrollable.vue:64]: also these comments? :)

File: packages/vue/src/components/molecules/SfScrollable/SfScrollable.vue [hunk 3]
```diff
@@ -64,22 +61,22 @@ export default {
...
+    //   observer.observe(this.contentEl, { childList: true });
+    // });
   },
   methods: {
-    sizeCalc() {
-      const containerHeight = this.$refs.content.$el.offsetHeight;
-      const contentHeight = this.contentEl.offsetHeight;
-      this.hasScroll = contentHeight > containerHeight;
-    },
+    // sizeCalc() {
```
Comment (justyna-13) [inline packages/vue/src/components/molecules/SfScrollable/SfScrollable.vue:75]: and here :)

**Review by justyna-13 — COMMENTED** [review]
see my comments :)

**Comment by justyna-13** [comment]
Please change hotfix branch to `hotfix-0.10.6-master` :slightly_smiling_face:

**Comment by Baroshem** [comment]
Closing as there was already a pull request to other hotfix branch

**Pull request closed without merging**